honeybee_schema/radiance/state.py

```diff
@@ -0,0 +1,78 @@
+"""State Schema"""
+
+from pydantic import Field, constr
+from typing import List
+from .._base import NoExtraBaseModel, IDdBaseModel
+
+from ..geometry import Face3D
+
+
+class StateGeometryAbridged(IDdBaseModel):
+    """StateGeometryAbridged represents shading for States."""
+
+    type: constr(regex='^StateGeometryAbridged$') = 'StateGeometryAbridged'
+
+    modifier: str = Field(
+        default=None,
+        description='A string for a Honeybee Radiance Modifier (default: None).'
+        )
+
+    modifier_blk: str = Field(
+        default=None,
+        description='A string for a Honeybee Radiance Modifier to be used '
+                    'in direct solar simulations and in isolation studies (assessing'
+                    'the contribution of individual objects) (default: None).'
+        )
+
+    geometry: Face3D = Field(
+        ...,
+        description='Planar Face3D for the geometry.'
+    )
+
+
+class _RadianceStateBaseAbridged(NoExtraBaseModel):
+    """Base model for abridge Radiance State Schema"""
+
+    modifier: str = Field(
+        default=None,
+        description='A Radiance Modifier identifier (default: None).'
+    )
+
+    modifier_direct: str = Field(
+        default=None,
+        description='A Radiance Modifier identifier (default: None).'
+    )
+
+
+class RadianceShadeStateAbridged(_RadianceStateBaseAbridged):
+    """RadianceShadeStateAbridged represents a single state for a dynamic Shade."""
+
+    type: constr(regex='^RadianceShadeStateAbridged$') = 'RadianceShadeStateAbridged'
+
+    shades: List[StateGeometryAbridged] = Field(
```

Comment:
I would move this to the `_RadianceStateBaseAbridged` class as it is shared by both.

Reply:
FYI: I realised that this leaves the RadianceStateBaseAbridged class with no unique properties, so just deleted the _RadianceStateBaseAbridged class and inherited the former for the RadianceSubFaceStateAbridged. This seems cleaner to me but let me know if it isn't as logically conceptually.